src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/EnvelopeCreator.java

```diff
@@ -148,11 +149,11 @@ public static Envelope envelopeNotified() {
         return envelope("SSCS", Status.NOTIFICATION_SENT);
     }
 
-    private static List<ScannableItem> scannableItems() {
+    public static List<ScannableItem> scannableItems(Iterator<String> dcnFeed) {
```

Comment:
How about below rather than passing iterator reference ?

```
 private static ScannableItem scannableItem(
        String dcn,
        OcrData ocr,
        DocumentType documentType,
        String documentSubtype
    ) {
        Instant timestamp = getInstant();

        return new ScannableItem(
            dcn,
            timestamp,
            "test",
            "test",
            "return",
            timestamp,
            ocr,
            dcn + ".pdf",
            "test",
            documentType,
            documentSubtype
        );
    }

```

Reply:
Let's see how it looks ;)